### PR TITLE
My second hw commit

### PR DESCRIPTION
**Pull request by Vidhya-eng**
New files

### CODE REVIEW

File: Lesson3.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<html>
```
Comment (aberezhk) [inline Lesson3.html]: Good job! Could you please add navigation and footer bars from index.html to this page as well?
Later we will link all pages in one project. So they all should have same header and footer.

File: Lesson3.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                answerCmp.textContent= "Oh that's sad 😞"
+            }
+            function awsomeButtonClick(event){
+                answerCmp.textContent = "Wow !! It's Awsome😍"
+            }
+
+            goodButtonCmp.addEventListener("click",goodButtonClick);
+            badButtonCmp.addEventListener("click",badButtonClick);
+            awsomeButtonCmp.addEventListener("click",awsomeButtonClick);
+
```
Comment (MrTim) [inline Lesson3.html:30]: not needed empty line.

File: Lesson3.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                answerCmp.textContent= "That's great 😃"
+            }
+            function badButtonClick(event){
+                answerCmp.textContent= "Oh that's sad 😞"
+            }
+            function awsomeButtonClick(event){
+                answerCmp.textContent = "Wow !! It's Awsome😍"
+            }
+
+            goodButtonCmp.addEventListener("click",goodButtonClick);
```
Comment (MrTim) [inline Lesson3.html]: it's better to put space after comma. That was separation looks better:
`            goodButtonCmp.addEventListener("click", goodButtonClick);`

File: Lesson3.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <button id="goodButton">Good😃</button>
+        <button id="badButton">Bad😞</button>
+        <button id ="awsomeButton">Awsome😍</button>
+        <p id ="answer"></p>
+        <script>
+            let goodButtonCmp = document.getElementById("goodButton");
+            let badButtonCmp =document.getElementById("badButton")
+            let awsomeButtonCmp = document.getElementById("awsomeButton")
+            let answerCmp = document.getElementById("answer");
+            function goodButtonClick(event){
```
Comment (MrTim) [inline Lesson3.html]: `function goodButtonClick(event) {` - space before curly bracket

File: Lesson3.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                answerCmp.textContent= "That's great 😃"
```
Comment (MrTim) [inline Lesson3.html]: do not omit `;` at the end of statement. Think about it as required to have.
Check your code for that missing semicolon.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
+<html>
+    <head>
+        <title>My Simple Base Page</title>
+        <link rel="stylesheet" type="text/css" href="style.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Allan|Asap+Condensed&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <header class="navigation" >
+                <button id="burgerBtn"><i class="fa fa-bars"></i></button>
```
Comment (MrTim) [inline index.html]: :+1:

File: index.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+                <nav> <a href >Home</a></nav>
+               <nav> <a href>About me</a></nav>
+               <nav class="base"> <a href="#index.html">Base Page</a></nav>
+
```
Comment (MrTim) [inline index.html:18]: not needed empty line. And below your using `<br/>` which is not needed. You should be able to achieve the same with css styles.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+        </div><br><br>
+        <footer class ="endDescription">
+            <p>Made by Vidhya for Redi : Frontend-Development2</p>
+        </footer>
+        <script>
+            let burgerBtnCmp = document.getElementById("burgerBtn");
+            function burgerBtnClick(event){
+               let name = prompt("Which is your favorite place in munich ?");
+               alert("Your Favorite place in Munich is " + name);
+
```
Comment (MrTim) [inline index.html]: Empty line

File: style.css [hunk 9]
```diff
@@ -0,0 +1,62 @@
+body{
+    margin : 0;
+}
+.navigation {
+    display: flex;
+    flex-direction: row;
+
```
Comment (MrTim) [inline style.css]: Empty line :) 
Check the code below for not needed empty lines.
I personally  do one empty line as separator between css classes. This increase readability of the code.

File: style.css [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+    overflow:auto;
+}
+.navigation a {
+    text-decoration:none;
+    color: white;
+}
+
+
+
+.Section{
```
Comment (MrTim) [inline style.css]: camelCase - should be `.section {`

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice work! 
there are some comment regarding styling - which is important as well. So please fix those comments.

**Review by aberezhk — APPROVED** [review]
All good. Please merge.

File: Lesson3.html (hunk 1 above)
Comment (Vidhya-eng) [reply Lesson3.html]: i didnt understand.Do you mean ,,,i should add a navigation bar and a footer to this lesson3.html

**Pull request merged**